xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -3104,7 +3104,6 @@ void CVideoPlayer::Pause()
   if (GetSpeed() == 0)
   {
     SetSpeed(1);
```

Comment:
@FernetMenta 
I'm a little bit concerned about this code, as it seems like it might be dead code right now. Should I remove it?